SignalRServiceBenchmarkPlugin/plugins/Plugin.Microsoft.Azure.SignalR.Benchmark/SignalRPlugin/MasterMethods/CollectStatistics.cs

```diff
@@ -18,7 +18,7 @@ public Task Do(IDictionary<string, object> stepParameters, IDictionary<string, o
             Log.Information($"Start to collect statistics...");
 
             // Get parameters
-            stepParameters.TryGetTypedValue(SignalRConstants.Interval, out double interval, Convert.ToDouble);
+            stepParameters.TryGetTypedValue(SignalRConstants.Interval, out long interval, Convert.ToInt64);
```

Comment:
Long/int is enough. The latency is not possible to be < 1ms when message go through network. I change the type of interval and duration to long, thanks. @ccic

Reply:
Good.